### ISSUE TITLE
Add cart documentation

### ISSUE THREAD

**Issue by jewzaam** [lightblue-platform/openshift-lightblue-cart#3]
User guide?  Link from the README.md in this repo.

**Comment by luan-cestari** [comment]
The outcome artifacts of this task, would they be these ones below?
https://github.com/lightblue-platform/openshift-lightblue-cart/blob/master/README.md
http://dev.docs.lightblue.io/develop_on_openshift/README.html
http://docs.lightblue.io/cookbook/install_lightblue_on_openshift.html

**Comment by jewzaam** [comment]
https://github.com/lightblue-platform/openshift-lightblue-cart/blob/master/README.md
This should like to the quickstart http://docs.lightblue.io/cookbook/quickstart.html

http://dev.docs.lightblue.io/develop_on_openshift/README.html
This is out of date.

http://docs.lightblue.io/cookbook/install_lightblue_on_openshift.html
A subset of the quickstart.  Need to update something in the quickstart with the list of URL's in earlier comment if they're not there already.


In short, I think the cart doc should reference our documentation and be very minimal.  Maybe keep in how to quickly deploy it but details on using it etc I think should be in a gitbook

**Comment by luan-cestari** [comment]
Can I move this to in_progress due the issues you mentioned?

**Comment by jewzaam** [comment]
@luan-cestari have PR's for the docs been merged?  I can't remember where this was given it's been a month since this was put in ready for review.

**Comment by luan-cestari** [comment]
You mean this PR https://github.com/lightblue-platform/openshift-lightblue-cart/pull/6/files , right? I can merge it. But the other points you mentioned, those would be other issues or may I consider part of this issue and that they will be fixed?

**Comment by jewzaam** [comment]
Let's make them new issues, only because the metadata mgmt app doesn't work in the cart yet:  https://github.com/lightblue-platform/openshift-lightblue-cart/issues/5

----- Original Message -----

**Comment by luan-cestari** [comment]
As the PR was already merged and the issues were created *, I will close this issue.

*:
https://github.com/lightblue-platform/lightblue-docs-user/issues/67
https://github.com/lightblue-platform/openshift-lightblue-cart/issues/10
https://github.com/lightblue-platform/lightblue-docs-overview/issues/10
